test/support/custom_modem.ex

```diff
@@ -15,10 +15,13 @@ defmodule VintageNetMobileTest.CustomModem do
 
     %RawConfig{
       raw_config
-      | files: [{"chatscript.#{ifname}", "Service provider is #{config.service_provider}"}]
+      | files: [{"chatscript.#{ifname}", "Service provider APNs are #{config.service_providers}"}]
```

Comment:
```suggestion
      | files: [{"chatscript.#{ifname}", "The service providers are #{config.service_providers}"}]
```

Nit, just since service provider maps can contain more than an APN.